@xen-orchestra/web/src/pages/host/[id]/network.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="host-network-view">
+    <div class="host-network-table-container">
+      <UiTitle type="h4" class="header">
```

Comment:
The class is not needed if no CSS is applied, also there is no `type` prop on this component:
```suggestion
      <UiTitle>
```